### ISSUE TITLE
pinger.Ping() always returns true

### ISSUE THREAD

**Issue by Peterkn2001** [bluemurder/esp8266-ping#11]
In the example sketch, the "Error during ping command." message is never seen, as pinger.Ping() always == true, even if the website/IP address is unreachable.

```cpp
// Ping technologytourist.com
  Serial.printf("\n\nPinging technologytourist.com\n");
  if(pinger.Ping("technologytourist.com") == false)
  {
    Serial.println("Error during ping command.");
  }
```
A similar issue was raised here: https://github.com/bluemurder/esp8266-ping/issues/2
which applied to version 1.0.0 and was supposedly fixed in version 1.1.0
In the 1.x versions the return value was meant to be 0 or -1 rather than true or false.
I tested version 1.1.0 and I couldn't get that to work correctly either.

I suspect that changes to the ESP core may be responsible for these issue, but I've not tested that theory.

**Comment by LordMyschkin** [comment]
Problem is persisting - this library seems to be dead....

**Comment by dineshgyl** [comment]
Yes, i think ping function always return true when you check for ip address.
